Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:8-alpine
+RUN mkdir /data && \
+	apk --no-cache add git && \
+	npm install phantomjs-prebuilt@2.1.13 && \
+	npm install -g resume-cli
+RUN npm install -g jsonresume-theme-elegant && \
```

Comment:
What's the logic behind this to separate npm install to be under two RUN command?